code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
NASA suits, and all unique hard suits are in with the suit/space line.... I'll check on the RD line now... but removing this and adding the /space/ line stops all the hundred bomber jackets etc being stored.